app/javascript/src/LoginPage/SignupPageWrapper.jsx

```diff
@@ -4,24 +4,40 @@ import React from 'react'
 import {createReactWrapper} from 'utilities/createReactWrapper'
 
 import {LoginPage} from '@patternfly/react-core'
-import {SignupForm} from 'LoginPage'
+import {SignupForm, FlashMessages} from 'LoginPage'
 
 import 'LoginPage/assets/styles/loginPage.scss'
 
 import brandImg from 'LoginPage/assets/images/3scale_Logo_Reverse.png'
 import PF4DownstreamBG from 'LoginPage/assets/images/PF4DownstreamBG.svg'
 
+type Errors = {
+  [string]: string[]
+}
 type Props = {
   user: {
     email: string,
     firstname: string,
     lastname: string,
-    username: string
+    username: string,
+    errors: Errors
   },
   name: string,
   path: string
 }
 
+const ErrorMessages = ({errors}: {errors: Errors}) => {
+  return Object.entries(errors).map(
```

Comment:
You could avoid this if from the beginning return the desired format, apparently, an Array. Then the `errors` in the state would be an Array of Objects instead of an Object that it was parsed to store it and then it needs to be parse (again) for using it